pulp_python/app/models.py

```diff
@@ -42,20 +61,20 @@ class PythonPackageContent(Content):
     name = models.TextField(blank=False)
     version = models.TextField(blank=False)
     metadata_version = models.TextField(blank=False)
-    summary = models.TextField()
-    description = models.TextField()
-    keywords = ArrayField(models.TextField())
-    home_page = models.TextField()
-    download_url = models.TextField()
-    author = models.TextField()
-    author_email = models.TextField()
-    maintainer = models.TextField()
-    maintainer_email = models.TextField()
-    license = models.TextField()
-    requires_python = models.TextField()
-    project_url = models.TextField()
-    platform = models.TextField()
-    supported_platform = models.TextField()
+    summary = models.TextField(null=True, max_length=8192)
+    description = models.TextField(null=True, max_length=8192)
+    keywords = models.TextField(null=True)
```

Comment:
For all these optional text fields, I think we want `blank=True`, not `null=True`.

https://docs.djangoproject.com/en/1.11/ref/models/fields/#null

Reply:
The issue is that some projects on pypi has the values set to null. 
E.g. `maintainer` in test pypi Django: https://test.pypi.org/pypi/Django/json